### PR TITLE
Disable events-004.html

### PR DESCRIPTION
**Pull request by jdm**
#13887

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/13922)

### CODE REVIEW

**Comment by emilio** [comment]
That being said, that test should come back as non-intermittent after #13864, I hope.

**Comment by KiChjang** [comment]
```
Tests with unexpected results:
  ▶ PASS [expected FAIL] /css21_dev/html4/c5525-fltblck-000.htm

  ▶ Unexpected subtest result in /css-transitions-1_dev/html/events-004.htm:
  │ FAIL [expected PASS] repeating lists / events
  │   → assert_equals: Expected TransitionEnd events triggered on .transition expected "padding-bottom:0.1s padding-left:0.05s padding-right:0.05s padding-top:0.1s" but got "padding-bottom:0.1s padding-bottom:0.1s padding-left:0.05s padding-right:0.05s padding-top:0.1s padding-top:0.1s"
  │ 
  │ root.generalParallelTest.assertExpectedEventsFunc/<@http://web-platform.test:8000/css-transitions-1_dev/html/support/generalParallelTest.js:207:13
  │ Test.prototype.step@http://web-platform.test:8000/resources/testharness.js:1227:20
  │ .cases.events.done@http://web-platform.test:8000/css-transitions-1_dev/html/events-004.htm:88:29
  │ runLoop/</</</<@http://web-platform.test:8000/css-transitions-1_dev/html/support/runParallelAsyncHarness.js:122:26
  │ Test.prototype.step@http://web-platform.test:8000/resources/testharness.js:1227:20
  │ runLoop/</</<@http://web-platform.test:8000/css-transitions-1_dev/html/support/runParallelAsyncHarness.js:121:21
  │ runLoop/</<@http://web-platform.test:8000/css-transitions-1_dev/html/support/runParallelAsyncHarness.js:120:17
  └ runLoop/<@http://web-platform.test:8000/css-transitions-1_dev/html/support/runParallelAsyncHarness.js:118:13
```

It's not disabled?

File: tests/wpt/metadata-css/css-transitions-1_dev/html/events-004.html.ini [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[events-004.html]
```
Comment (KiChjang) [inline tests/wpt/metadata-css/css-transitions-1_dev/html/events-004.html.ini]: Ugh, this is NOT a `.html` file, instead it's a `.htm` file.

**Comment by KiChjang** [comment]
@bors-servo retry
* #13920

**Comment by KiChjang** [comment]
@bors-servo retry
* #13920

**Pull request merged**